spec/wizards/placements/add_placement_wizard/terms_step_spec.rb

```diff
@@ -35,7 +35,7 @@
 
     it "returns mentors for the school, ordered by name" do
```

Comment:
Ditto – this looks like a copy-pasta

```suggestion
    it "returns all terms in order" do
```